cocos/core/utils/misc.ts

```diff
@@ -179,8 +178,13 @@ export function callInNextTick (callback, p1?: any, p2?: any): void {
  * @zh 被 try catch 包裹的函数名。
  * @returns @en A new function that will invoke `functionName` with try catch.
  * @zh 使用 try catch 机制调用 `functionName` 的新函数.
+ *
+ * @deprecated `misc.tryCatchFunctor_EDITOR` is deprecated since v3.9.0.
  */
-export function tryCatchFunctor_EDITOR (funcName: string): (comp: Component) => void {
+export function tryCatchFunctor_EDITOR (funcName: string): (comp: unknown) => void {
+    if (DEBUG) {
+        warnID(16000, 'misc.tryCatchFunctor_EDITOR', '3.9.0');
+    }
```

Comment:
The warning message doesn't say it is deprecated.

Reply:
it is metioned here 
https://github.com/cocos/cocos-engine/blob/develop/EngineErrorMap.md#16000